src/Workleap.DomainEventPropagation.Abstractions/IDomainEventWrapper.cs

```diff
@@ -0,0 +1,20 @@
+namespace Workleap.DomainEventPropagation;
+
+public interface IDomainEventWrapper
```

Comment:
I think this should be named `IDomainEvent` without `Wrapper` now that it is public